tests/Requirement_UnitTest.py

```diff
@@ -96,7 +96,7 @@ def test_Construct():
 # ----------------------------------------------------------------------
 def test_Success():
     requirement = MyRequirement(Mock(), Mock(), Mock(), Mock(), Mock(), EvaluateResult.Success, "testing")
-    result_info = requirement.Evaluate(Mock(), {})
+    result_info = requirement.Evaluate({"key": "value"}, {})

```

Comment:
@davidbrownell I can replace `{"key": "value"}` with `{}` as well if you'd like.

Reply:
It depends on whether the intent of the test is to exercise the default scenario or the scenario where the value is overridden.

Reply:
I feel this is more descriptive so I would prefer to keep it in.